shared/src/main/scala/com/raquo/domtypes/generic/builders/EventPropBuilder.scala

```diff
@@ -1,10 +1,10 @@
 package com.raquo.domtypes.generic.builders
 
-import com.raquo.domtypes.generic.keys.EventProp
+/**
+  * @tparam EP EventProp
+  * @tparam DomEvent Base type for DOM events (dom.Event in Scala JS)
+  */
+trait EventPropBuilder[EP[Ev <: DomEvent], DomEvent] {
 
-trait EventPropBuilder extends SpecializedBuilder[EventProp] {
-
-  @inline override def build[Ev](key: String): EventProp[Ev] = {
-    new EventProp[Ev](name = key)
-  }
+  @inline def build[Ev <: DomEvent](key: String): EP[Ev]
```

Comment:
Maybe rename `build` to `events` to have similar naming like the other builders?

Reply:
Oh definitely, good catch, thanks!

Reply:
I think I'll name it `eventProp` though for consistency, and, well – to be truthful – since it canonically returns an `generic.keys.EventProp`, just like the `attr` method canonically returns an `generic.keys.Attr`.